src/components/IconRounded/WIconRounded.vue

```diff
@@ -13,9 +13,13 @@ const props = defineProps({
     type: Number,
     required: false,
     default: 70
+  },
+  disableHoverEffect: {
```

Comment:
The hover effect is removing the background and `disasbleHoverEffect` doesn't disable hover.
![image](https://github.com/user-attachments/assets/f0d8ee48-897f-43c5-b1e8-3be57787e8be)
![image](https://github.com/user-attachments/assets/1e9850ef-210e-4a95-9128-abeae21f1732)